torch/csrc/CudaIPCTypes.cpp

```diff
@@ -0,0 +1,156 @@
+#ifdef USE_CUDA
+#include <torch/csrc/CudaIPCTypes.h>
```

Comment:
Wondering if this shouldn't go in the `torch/csrc/cuda` folder. I'm not too familiar with how the build works here, but it seems worth looking into, or maybe asking @zdevito about